debian/rules

```diff
@@ -22,6 +22,8 @@ KERNEL_TYPE.esx := generic
 KERNEL_TYPE.gcp := gcp
 KERNEL_TYPE.kvm := kvm
 
+SIGN_KEY_TYPES = registration upgrade
```

Comment:
To match the `DELPHIX_SIGNATURE_VERSION` variable name, can we change to to something like `DELPHIX_SIGNATURE_TYPES` to be a little more consistent?

Reply:
Sounds good, will do.